.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,3 +9,24 @@ Please make sure you provide the relevant context.
 __Which issue does this PR address?__
 
 Closes #
+
+__Acceptance Criteria__
+
+<!--
+
+Link the acceptance criteria here if they are defined.
+
+-->
+
+* [ ] Corresponds to the concept <!-- link document here -->
+* [ ] Corresponds to the design <!-- link document here -->
+
+__Definition of Done__
+
+* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
+* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
+* [ ] passes Continuos Integration checks
```

Comment:
Continuos ➡️ Continuous

Reply:
Lol, I used spelling suggested by the typo checker extension :D